at_secondary/at_persistence_secondary_server/lib/src/log/commitlog/commit_log_keystore.dart

```diff
@@ -341,8 +341,7 @@ class CommitLogKeyStore
     if ((RegExp(regex).hasMatch(atKey)) ||
         atKey.contains(AT_ENCRYPTION_SHARED_KEY) ||
```

Comment:
Should we refactor this to:

match =  _isRegexMatches() || _isSpecialKey();

_isSpecialKey contains checking for special keys

Reply:
yeah..sounds good.i will make the changes